driver/socketcall_to_syscall.c

```diff
@@ -21,32 +22,32 @@ or GPL2.txt for full copies of the license.
 /* We need to import them explicitly otherwise if we import `linux/net`
  * we will have a redefinition of syscall codes.
  */
-#define SYS_SOCKET	1		/* sys_socket(2)		*/
-#define SYS_BIND	2		/* sys_bind(2)			*/
-#define SYS_CONNECT	3		/* sys_connect(2)		*/
-#define SYS_LISTEN	4		/* sys_listen(2)		*/
-#define SYS_ACCEPT	5		/* sys_accept(2)		*/
-#define SYS_GETSOCKNAME	6		/* sys_getsockname(2)		*/
-#define SYS_GETPEERNAME	7		/* sys_getpeername(2)		*/
-#define SYS_SOCKETPAIR	8		/* sys_socketpair(2)		*/
-#define SYS_SEND	9		/* sys_send(2)			*/
-#define SYS_RECV	10		/* sys_recv(2)			*/
-#define SYS_SENDTO	11		/* sys_sendto(2)		*/
-#define SYS_RECVFROM	12		/* sys_recvfrom(2)		*/
-#define SYS_SHUTDOWN	13		/* sys_shutdown(2)		*/
-#define SYS_SETSOCKOPT	14		/* sys_setsockopt(2)		*/
-#define SYS_GETSOCKOPT	15		/* sys_getsockopt(2)		*/
-#define SYS_SENDMSG	16		/* sys_sendmsg(2)		*/
-#define SYS_RECVMSG	17		/* sys_recvmsg(2)		*/
-#define SYS_ACCEPT4	18		/* sys_accept4(2)		*/
-#define SYS_RECVMMSG	19		/* sys_recvmmsg(2)		*/
-#define SYS_SENDMMSG	20		/* sys_sendmmsg(2)		*/
-
-/* Please note that here `socketcall_syscall_id` could be the code
- * on 64-bit or 32-bit.
- */
-int socketcall_code_to_syscall_code(int socketcall_code)
+#define SYS_SOCKET 1	  /* sys_socket(2)		*/
+#define SYS_BIND 2	  /* sys_bind(2)			*/
+#define SYS_CONNECT 3	  /* sys_connect(2)		*/
+#define SYS_LISTEN 4	  /* sys_listen(2)		*/
+#define SYS_ACCEPT 5	  /* sys_accept(2)		*/
+#define SYS_GETSOCKNAME 6 /* sys_getsockname(2)		*/
+#define SYS_GETPEERNAME 7 /* sys_getpeername(2)		*/
+#define SYS_SOCKETPAIR 8  /* sys_socketpair(2)		*/
+#define SYS_SEND 9	  /* sys_send(2)			*/
+#define SYS_RECV 10	  /* sys_recv(2)			*/
+#define SYS_SENDTO 11	  /* sys_sendto(2)		*/
+#define SYS_RECVFROM 12	  /* sys_recvfrom(2)		*/
+#define SYS_SHUTDOWN 13	  /* sys_shutdown(2)		*/
+#define SYS_SETSOCKOPT 14 /* sys_setsockopt(2)		*/
+#define SYS_GETSOCKOPT 15 /* sys_getsockopt(2)		*/
+#define SYS_SENDMSG 16	  /* sys_sendmsg(2)		*/
+#define SYS_RECVMSG 17	  /* sys_recvmsg(2)		*/
+#define SYS_ACCEPT4 18	  /* sys_accept4(2)		*/
+#define SYS_RECVMMSG 19	  /* sys_recvmmsg(2)		*/
+#define SYS_SENDMMSG 20	  /* sys_sendmmsg(2)		*/
+
+int socketcall_code_to_syscall_code(int socketcall_code, bool* is_syscall_return)
```

Comment:
This feels weird; still, we will need to come back at this impl later, therefore no blocker for me.